src/ledctl/ledctl.c

```diff
@@ -406,8 +406,10 @@ static led_status_t _ibpi_state_add_block(struct ibpi_state *state, char *name)
 	char path[PATH_MAX];
 	led_status_t rc = led_device_name_lookup(ctx, name, path);
 
-	if (rc != LED_STATUS_SUCCESS)
+	if (rc != LED_STATUS_SUCCESS) {
+		log_error("could not find %s, make sure device exists", name);
 		return rc;
```

Comment:
I think that it should start with upper letter and end with "."
I know that we are not consistent in messages style (maybe we should add test for that?) but messages in main are in "Message." form (some of them were recently added).
I think that we should keep this form for new messages.

Also, I see "make sure device exists" an unnecessary, it is kind of waste.
I mean that "Could not find %s" is same in understand (first you need to check if this device exists), we don't need to give user suggestion in this case but if you see some advantages, let me know. I'm always open to change my mind.

Reply:
My reason for "make sure device exists" is: "provide informative messages that guide users towards solutions." 😄

Reply:
Maybe "Device %s does not exist" then?